pkg/sync/options.go

```diff
@@ -0,0 +1,23 @@
+package sync
+
+import "time"
+
+type Options struct {
+	Interval   time.Duration
+	URL        string
+	Name       string
+	Namespace  string
+	Branch     string
+	TargetPath string
+}
```

Comment:
Should there be an option to generate bucket manifests?

Reply:
No since bootstrap only works with Git.